pages/[id].js

```diff
@@ -10,13 +10,28 @@ export default function Activity() {
   const activity = dummyData.find((activity) => activity.id === id);
 
   return (
-    <>
+    <main>
```

Comment:
If you're planning to wrap all pages in ```<main></main>```you can also add this to the highest level around the components.

Reply:
Thank you, we tried this before, we choose the current solution to avoid double-paddings. We will look at this later on, when we are working on Styling-Details.